word_guess.rb

```diff
@@ -0,0 +1,177 @@
+require 'faker'
+
+#creates array with 10 random names. partitions first name of each character
+simpsons = []
+10.times do
+  name = Faker::Simpsons.character.downcase
+  name = name.partition(" ").first
+  simpsons << name
+end
+
+class Game
```

Comment:
The `Game` class doesn't seem to have much `Game` logic in it.  Most of the game logic rests in `Word`, maybe looking at what roles each class is performing would be worthwhile.